app/actors/DeliveryJournalActor.scala

```diff
@@ -2,32 +2,41 @@ package actors
 
 import akka.actor.{ActorSystem, Cancellable}
 import play.api.db.Database
+import service.DeliveryEventConsumer
+import software.amazon.awssdk.auth.credentials.{AwsBasicCredentials, StaticCredentialsProvider}
+import software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClient
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.kinesis.KinesisAsyncClient
 
 import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
 import scala.concurrent.duration.{FiniteDuration, SECONDS}
 import scala.util.{Success, Try}
 
 @Singleton
-class DeliveryJournalActor @Inject()(system: ActorSystem, override val db: Database)
+class DeliveryJournalActor @Inject()(system: ActorSystem,
+                                     deliveryEventConsumer: DeliveryEventConsumer,
+                                     override val db: Database)
   extends DBPollActor(table = "deliveries") {
 
   println("actor initialized")
 
   override def preStart(): Unit = {
     // super.preStart()
     log.info("[DeliveryJournalActor] Inside preStart")
-    //    self ! "Insert"
+    //        self ! "Insert"
+    startPolling()
   }
 
   def schedule(): Cancellable = {
-    system.scheduler.scheduleWithFixedDelay(FiniteDuration(5, SECONDS), delay, self, "Insert")(system.dispatcher)
+    system.scheduler.scheduleWithFixedDelay(FiniteDuration(5, SECONDS), delay, self, "INSERT")(system.dispatcher)
   }
 
   override def process(record: ProcessQueueDelivery): Try[Unit] = {
     record.operation match {
-      case "INSERT" | "UPDATE" => // TODO: Publish using kinesis
+      case "INSERT" | "UPDATE" =>
         Try {
```

Comment:
I think we don't need Try as the method calling `process` method is already wrapped inside Try block. Remove it and change the return type to Unit. Also, maintain the consistency with order service. [look at this](https://github.com/prakharsingh2818/order/pull/8/files#diff-46437fa4f87bfd64b7557d8e9dba9cab8f014ec420aa336c74647a98e2c0ffedR27)